drivers/SmartThings/matter-sensor/fingerprints.yml

```diff
@@ -117,6 +117,11 @@ matterGeneric:
     deviceTypes:
       - id: 0x002C
     deviceProfileName: air-quality-sensor
+  - id: "matter/air-quality-ikea"
+    deviceLabel: IKEA air Quality Sensor
+    deviceTypes:
+      - id: 0x002C # Air Quality
```

Comment:
We already have a fingerprint for this device `matter/air-quality-sensor` so I don't think we want to add a new fingerprint. Instead we should detect the clusters at runtime and then profile switch to the profile.

Reply:
Removed the fingerprint and made the recommended updates in 2f6d6be9343a806c29c8f9bfaa5122623aa0d116.